appengine/standard_python3/bundled-services/deferred/wsgi/main_test.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import subprocess
+import time
+
+import pytest
+import requests
+
+
+@pytest.fixture
+def version():
+    """Launch a new version of the app for testing, and yield the
+    project and version number so tests can invoke it, then delete it.
+    """
+
+    output = subprocess.run(
+        "gcloud app deploy --no-promote --quiet --format=json",
+        capture_output=True,
+        shell=True,
+    )
+
+    result = json.loads(output.stdout)
+    version_id = result["versions"][0]["id"]
```

Comment:
Print out `output.stdout` and `output.stderr` for debuggability.

Reply:
Done